tox.ini

```diff
@@ -40,7 +40,7 @@ norecursedirs = .* docs requirements site-packages
 deps =
     -r{toxinidir}/requirements/test.txt
 commands =
-    pytest {posargs}
```

Comment:
what about this change? :thinking:

Reply:
Explicitly mentioning the target directories is a standard practice being followed across edX/OpenedX repos so I updated it accordingly.

Reply:
Good! Thanks!